docs/articles/new-expensify/account-settings/Preferences.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Preferences
+description: How to manage your Expensify Preferences
+---
+# Overview
+Your Preferences in Expensify allow you to customise how you use New Expensify.
+
+- Set your theme preference
+
+# How to set your theme preference in New Expensify
+
+To set or update your theme preference in New Expensify: 
+1. Go to **Settings > Preferences**
+2. Tap on **Theme**
+3. You can choose between the _Dark_ theme, the _Light_ theme, or _Use Device Settings_ (which is the default setting)
+
+_Use Device Settings_ is the default setting.
```

Comment:
NAB but this is kind of redundant, thoughts on dropping the `(which is the default setting)`?

Reply:
Yep, good point!